test/fixtures/uncaught-exceptions/callbackify2.js

```diff
@@ -8,13 +8,13 @@ const { callbackify } = require('util');
 {
   const sentinel = new Error(__filename);
   process.once('uncaughtException', (err) => {
-    assert.strictEqual(err, sentinel);
+    assert.notStrictEqual(err, sentinel);
     // Calling test will use `stdout` to assert value of `err.message`
     console.log(err.message);
   });
 
   async function fn() {
-    return await Promise.reject(sentinel);
+    return Promise.reject(sentinel);
```

Comment:
if you remove `await`  there is no need for this to be an async function.

Reply:
Hm, true. I am not sure why it was a async function in the first place. I guess a oversight.